src/cache/cache_las.c

```diff
@@ -958,8 +1030,9 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
 		}
 
 		WT_ERR(cursor->remove(cursor));
-		++remove_cnt;
+		++decrement_cnt;
 	}
+
```

Comment:
Remove whitespace

Reply:
Why? The locking call isn't related to the `for` loop in any way, and the lock call isn't near at the start of the `for` loop, so think they should be separate.